Makefile

```diff
@@ -117,3 +121,12 @@ test-e2e: image hyperkube
 
 test-ccm-e2e:
 	go test ./tests/e2e/ -timeout 0 -v $(CCM_E2E_ARGS)
+
+deploy: image hyperkube
+        docker build $(CCM_IMAGE)\
+            --build-arg K8S_VERSION=$(K8S_VERSION) \
+		    --build-arg AKSENGINE_VERSION=$(AKSENGINE_VERSION) \
+		    tests/k8s-azure
+		docker push $(CCM_IMAGE)
+		touch	TEMPFILE	|	sed	's/$(IMAGE)/$(CCM_IMAGE)/'	/tests/k8s-azure/manifests/$(TEST_MANIFEST)	>TEMPFILE	\
```

Comment:
we can move the file changes to hack/deploy-cluster.sh and let the script accept $(IMAGE)